polly/lib/CodeGen/IRBuilder.cpp

```diff
@@ -128,8 +128,28 @@ void ScopAnnotator::popLoop(bool IsParallel) {
   LoopAttrEnv.pop_back();
 }
 
-void ScopAnnotator::annotateLoopLatch(BranchInst *B, Loop *L, bool IsParallel,
-                                      bool IsLoopVectorizerDisabled) const {
+void ScopAnnotator::addVectorizeMetadata(LLVMContext &Ctx,
```

Comment:
```suggestion
static void addVectorizeMetadata(LLVMContext &Ctx,
```
AFICS this does not use any `ScopAnnotator` members